apps/camera_pipe/CMakeLists.txt

```diff
@@ -8,8 +8,9 @@ set(curved_lib "${CMAKE_CURRENT_BINARY_DIR}/curved${CMAKE_STATIC_LIBRARY_SUFFIX}
 add_library(fcam fcam/Demosaic.cpp fcam/Demosaic_ARM.cpp)
 FIND_PACKAGE(OpenMP)
 if (OPENMP_FOUND)
-    target_compile_options(fcam PUBLIC "-fopenmp")
+  target_compile_options(fcam PUBLIC "-fopenmp")
 endif()
+target_compile_options(fcam PRIVATE "-std=c++11")
```

Comment:
Later down in the file, there is an MSVC conditional around similar logic. Is it needed here?